pkg/systemd/overview-cards/cryptoPolicies.jsx

```diff
@@ -0,0 +1,230 @@
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2022 Red Hat, Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import cockpit from "cockpit";
+import React, { useState, useEffect } from 'react';
+import { Button, Flex, Modal, Popover } from "@patternfly/react-core";
+import { ExternalLinkSquareAltIcon, HelpIcon, InProgressIcon } from '@patternfly/react-icons';
+
+import { ModalError } from 'cockpit-components-inline-notification.jsx';
+import { ShutdownModal } from 'cockpit-components-shutdown.jsx';
+import { PrivilegedButton } from "cockpit-components-privileged.jsx";
+import { TunedDialogBody } from '../../../pkg/tuned/change-profile.jsx';
+
+import "./cryptoPolicies.scss";
+
+const _ = cockpit.gettext;
+
+const cryptopolicies = {
```

Comment:
Please add a comment where you took the descriptions from, so that we can update them occasionally.